linux-audio/fluidsynth2-static.json

```diff
@@ -2,16 +2,18 @@
   "name": "fluidsynth",
   "buildsystem": "cmake-ninja",
   "build-options": {
-    "cflags": "-fPIC",
-    "cxxflags": "-fPIC"
+    "env": {
```

Comment:
This overrides default CFLAGS/CXXFLAGS variables. Don't do this. How this change is even related to this PR?

Reply:
I literally just grabbed the file from commit https://github.com/flathub/shared-modules/commit/a79072c2f6cd6c5b1a6cb8ccec649cfb87608926 and replaced the fluidsynth version, I didn't make any more changes.